app/models/reports/trainee_report.rb

```diff
@@ -298,12 +298,12 @@ def last_names
       trainee.last_name
     end
 
-    def lead_school_name
-      trainee.lead_partner_not_applicable? ? I18n.t(:not_applicable) : trainee.lead_school&.name
+    def lead_partner_name
+      trainee.lead_partner_not_applicable? ? I18n.t(:not_applicable) : trainee.lead_partner&.name
     end
 
-    def lead_school_urn
-      trainee.lead_school&.urn
+    def lead_partner_urn
+      trainee.lead_partner&.urn
```

Comment:
This won't display anything for HEI LPs at the moment, but we're ok with that. At some point I think we'll want to add URNs to HEI LPs or display their UKPRNs instead.